### ISSUE TITLE
F-Droid can't build

### ISSUE THREAD

**Issue by licaon-kter** [Ruan625Br/FileManagerSphere#44]
ref: https://monitor.f-droid.org/builds/log/com.etb.filemanager/4#site-footer

/LE: fyi https://gitlab.com/fdroid/fdroiddata/-/commit/618a00470b6079c5e57626147c273912445b494d

**Comment by licaon-kter** [comment]
diff log: [etb4.log](https://github.com/Ruan625Br/FileManagerSphere/files/13164443/etb4.log)

local APK: [com.etb.filemanager_4.apk.zip](https://github.com/Ruan625Br/FileManagerSphere/files/13164446/com.etb.filemanager_4.apk.zip) _(remove .ZIP)_

upstream APK: https://github.com/Ruan625Br/FileManagerSphere/releases/tag/v1.1.2

@Ruan625Br can run run gradle clean or clean cache or whatever you can and rebuild a test APK from https://github.com/Ruan625Br/FileManagerSphere/commit/057226fcbbf6a38c7bdd3cf90f94e91dd85127fc ?

@linsui @obfusk fyi

**Comment by linsui** [comment]
You just need to update the apk.

**Comment by Ruan625Br** [comment]
I updated the apk

**Comment by licaon-kter** [comment]
Looks like I missed this, so...
```
Binary files /tmp/tmp3y7uxnhj/unsigned_binaries_com.etb.filemanager_4.binary/content/AndroidManifest.xml and /tmp/tmp3y7uxnhj/unsigned_com.etb.filemanager_4/content/AndroidManifest.xml differ
diff -r /tmp/tmp3y7uxnhj/unsigned_binaries_com.etb.filemanager_4.binary/content/META-INF/androidx.core_core-ktx.version /tmp/tmp3y7uxnhj/unsigned_com.etb.filemanager_4/content/META-INF/androidx.core_core-ktx.version
1c1
< 1.13.0-alpha01
---
> 1.13.0-alpha02
diff -r /tmp/tmp3y7uxnhj/unsigned_binaries_com.etb.filemanager_4.binary/content/META-INF/androidx.core_core.version /tmp/tmp3y7uxnhj/unsigned_com.etb.filemanager_4/content/META-INF/androidx.core_core.version
1c1
< 1.13.0-alpha01
---
> 1.13.0-alpha02
```

@Ruan625Br are you sure your apk in https://github.com/Ruan625Br/FileManagerSphere/releases/tag/v1.1.2 is built from https://github.com/Ruan625Br/FileManagerSphere/commit/057226fcbbf6a38c7bdd3cf90f94e91dd85127fc ?

**Comment by Ruan625Br** [comment]
yes

**Comment by licaon-kter** [comment]
Why was this closed?

**Comment by Ruan625Br** [comment]
sorry

**Comment by licaon-kter** [comment]
https://gitlab.com/fdroid/fdroiddata/-/jobs/7229820380#L1433

**Comment by linsui** [comment]
@Ruan625Br We need the api key to build the apk. Can you provide the key or a flavor without the key?

**Comment by Ruan625Br** [comment]
@linsui Which api?

**Comment by linsui** [comment]
Looks like it's for gemini-pro.

**Comment by Ruan625Br** [comment]
generate one for you on the gemini website

**Comment by linsui** [comment]
No, we can't use our own key. It's reproducible build so we need to use the same key.

**Comment by licaon-kter** [comment]
There's no APK for 1.1.3 ? @Ruan625Br
